src/Workspaces/Core/Portable/Shared/Extensions/IComparerExtensions.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
+
```

Comment:
Note: IComparer in corefx annotates the Compare method as [AllowNull] on the inputs -- this file would start to give warnings against a corefx annotated version I think.

Reply:
I'm OK leaving this for now. The implementation of `InverseComparer<T>` is correct in its current form for both nullable types and non-nullable types.